packages/widgets/user-profile-widget/jest.config.js

```diff
@@ -11,8 +11,8 @@ module.exports = {
     global: {
       branches: 0,
       functions: 12,
-      lines: 38,
-      statements: 37,
+      lines: 36,
+      statements: 36,
```

Comment:
Can we add a test for the API call?

Reply:
You marked it as resolved but it seems there are no new tests

Reply:
as there is not API :)